src/modules/tools/zprobe/ZProbe.h

```diff
@@ -10,9 +10,14 @@
 
 #include "Module.h"
 #include "Pin.h"
+#include "wait_api.h"
```

Comment:
this would go in the cpp file not the header. but as delays are not allowed it should be removed.